opencog/haskell/AtomSpace_CWrapper.h

```diff
@@ -0,0 +1,20 @@
+
+#include <opencog/atomspace/AtomSpace.h>
+
+extern "C"
+{
+    using namespace opencog;
+
+    AtomSpace* AtomSpace_new();
+    void AtomSpace_delete(AtomSpace* this_ptr);
+    long AtomSpace_addNode(AtomSpace* this_ptr, Type t, const char* name);
+/*
+    int AtomSpace_getSize(AtomSpace* this_ptr);
+    int AtomSpace_getNumNodes(AtomSpace* this_ptr);
+    int AtomSpace_getNumLinks(AtomSpace* this_ptr);
+    void AtomSpace_addAtom(AtomSpace* this_ptr, AtomPtr atom);
+    Handle AtomSpace_addLink(AtomSpace* this_ptr, Type t, const HandleSeq& outgoing);
+    bool AtomSpace_removeAtom(AtomSpace* this_ptr, Handle h);
+*/
+}
+
```

Comment:
I would assume that a C wrapper for the AtomSpace should be moved to its own directory, something like
```
opencog/atomspace/C-wrapper
```
any suggestion, guys?

Anyway for now it's OK to let this under opencog/haskell